processing/data_collection/gazette/spiders/mg_itauna.py

```diff
@@ -0,0 +1,36 @@
+import datetime as dt
+
+from dateparser import parse
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgItaunaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3133808"
+    name = "mg_itauna"
+    allowed_domains = ["itauna.mg.gov.br"]
+    start_urls = ["https://www.itauna.mg.gov.br/portal/diario-oficial/"]
+
+    def parse(self, response):
+        diarios = response.xpath("//div[@class='d_e_modelo_diario']")
+        for diario in diarios:
+            url = (
+                "https://www.itauna.mg.gov.br"
+                + diario.css("a")[1].extract().split('"')[1]
```

Comment:
As a tip, these kinds of selections can break easily if the website adds one element `a` before the one you are expecting or just inverting the order of the links.

One thing you could do in this case is look for a pattern that these links have and use it.

An example is this xpath `.//a[contains(@href, "downloads")]/@href` which only retrieves the `href` attribute of `a` elements which contains the string "downloads" in it's `href` attribute.

Reply:
As another tip, you can use `::attr()` (in CSS selectors) to specify which attribute you want to extract. That way you wouldn't need to split by `"` and select the first attribute.

Reply:
Maybe using `response.urljoin(href)` instead of `"https://www.itauna.mg.gov.br" + href` could be better since you don't need to hardcode the base url?